### PR TITLE
Extended simulation tutorial with bootstrap server

### PR DESCRIPTION
**Pull request by devos50**
Fixes #(issue number)

This PR:

 - Extends the simulation tutorial with a simple example of how to leverage the IPv8 bootstrap server.

### CODE REVIEW

**Comment by devos50** [comment]
validate

File: ipv8/messaging/interfaces/endpoint.py [hunk 1]
```diff
@@ -129,6 +129,7 @@ def __init__(self, endpoint, main_thread=True):
         self.endpoint = endpoint
 
         self._netifaces_failed = netifaces is None
+        self._local_interfaces = []
```
Comment (qstokkink) [inline ipv8/messaging/interfaces/endpoint.py:132]: This was already included in #999. Generally: please make sure to only make one change per PR.

**Review by qstokkink — CHANGES_REQUESTED** [review]

File: ipv8/messaging/interfaces/endpoint.py (hunk 1 above)
Comment (devos50) [reply ipv8/messaging/interfaces/endpoint.py:132]: Oh whoops, this one slipped through I guess 🤔

**Comment by devos50** [comment]
validate

**Review by qstokkink — APPROVED** [review]

**Comment by qstokkink** [comment]
Also, congrats on netting PR #1000 😄

**Pull request merged**